cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
@kkrentz #1737 Fixes this reboot issue for me. Can you please confirm that it works for your use cases too? If so, you can drop the first commit of this PR (cf9fece27a9475f064252cefb740248ce21a6488), and then I can merge it.